lms/djangoapps/django_comment_client/tests/test_utils.py

```diff
@@ -1345,42 +1345,52 @@ def to_id(name):  # pylint: disable=missing-docstring
         config_course_cohorts(
             course,
             is_cohorted=True,
+        )
+        config_course_discussions(
+            course,
             discussion_topics=["General", "Feedback"],
             divided_discussions=["Feedback", "random_inline"]
         )
+
         self.assertFalse(
             utils.is_commentable_divided(course.id, to_id("random")),
             "By default, Non-top-level discussions are not cohorted in a cohorted courses."
         )
 
-        # if always_cohort_inline_discussions is set to False, non-top-level discussion are always
+        # if always_divide_inline_discussions is set to False, non-top-level discussion are always
```

Comment:
Might as well update the rest of this comment for the new variable names (not divided unless... divided_discussions).